src/contexts/Api/index.tsx

```diff
@@ -184,6 +196,7 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
       newApi.consts.staking.historyDepth,
```

Comment:
separate chainState and consts into 2 separate functions, & fetch consts in a useEffect once chainState has been fetched.